src/measure/quant.rs

```diff
@@ -13,6 +13,12 @@ impl Encodable for Quant {
     }
 }
 
+impl Decodable for Quant {
+    fn decode<D: Decoder>(d: &mut D) -> Result<Self, D::Error> {
+        d.read_u32().map(|q| Quant(q))
+    }
+}
+
```

Comment:
Make a little tests that encodes quant, decodes it and assert that the result didn't change.